### PR TITLE
prepare v0.1.6 for release [skip vbump]

### LINKED ISSUE #118: release (0.1.6) ready by 13-March-2023
- [x] increase versioned dependency on scda.2022 to >=0.1.4, blocked by https://github.com/insightsengineering/scda.2022/issues/108
- [x]  Bump version to 0.1.6
- [x]  check and update README and NEWS.md
- [ ]  tag github release 0.1.6
- [ ]  update chevron DESCRIPTION of scda >= 0.1.6

### PR DESCRIPTION
**Pull request by ayogasekaram**
closes #118

### CODE REVIEW

**Comment by ayogasekaram** [comment]
merge https://github.com/insightsengineering/scda.2022/pull/110 before.

File: NEWS.md [hunk 1]
```diff
@@ -1,35 +1,25 @@
-# scda 0.1.5.9009
+# scda 0.1.6
 
-* Updated examples to use `scda.2022` data.
 * Remove dependency on `scda.2021`.
 
 # scda 0.1.5
```
Comment (edelarua) [inline NEWS.md:5]: Same as `scda.2022` - I would leave older sections as is.

File: NEWS.md [hunk 2]
```diff
@@ -1,35 +1,25 @@
-# scda 0.1.5.9009
+# scda 0.1.6
 
-* Updated examples to use `scda.2022` data.
```
Comment (edelarua) [inline NEWS.md:3]: I would leave this in too.

**Review by edelarua — CHANGES_REQUESTED** [review]
Same comments as the other PR - otherwise looks fine!

**Review by edelarua — APPROVED** [review]
Lgtm!

**Pull request merged**